### PR TITLE
Index operation

### PR DESCRIPTION
**Pull request by xluo-aws**
### Description
[Remove split functions from IndicesActions class; Minor wording change after review; ]

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Hailong-am — APPROVED** [review]

File: public/pages/SplitIndex/components/SplitIndexFlyout/SplitIndexFlyout.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default class SplitIndexFlyout extends Component<SplitIndexComponentProps
       {
         rowProps: {
           label: "Target Index Name",
+          helpText: `Must be in lowercase letters. Cannot begin with underscores or hyphens. Spaces, commas, and characters :, ", *, +, /, \, |, ?, #, > are not allowed.`,
```
Comment (Hailong-am) [inline public/pages/SplitIndex/components/SplitIndexFlyout/SplitIndexFlyout.tsx:55]: can we have this help text in common place and reuse for create/shrink index page?

File: public/pages/SplitIndex/container/SplitIndex/SplitIndex.tsx [hunk 2]
```diff
@@ -43,6 +44,7 @@ export class SplitIndex extends Component<SplitIndexProps> {
   };
 
   async componentDidMount() {
+    this.context.chrome.setBreadcrumbs([BREADCRUMBS.INDEX_MANAGEMENT, BREADCRUMBS.INDICES, BREADCRUMBS.SPLIT_INDEX]);
```
Comment (SuZhou-Joe) [inline public/pages/SplitIndex/container/SplitIndex/SplitIndex.tsx:47]: The last path will be a dynamic breadcrumb, it's recommended to add current path with dynamic params as the last breadcrumb, which should be { ...BREADCRUMBS.SPLIT_INDEX, path: '${your_path_and_queries_to_split_index}' }

**Review by Hailong-am — APPROVED** [review]

File: public/pages/SplitIndex/components/SplitIndexFlyout/SplitIndexFlyout.tsx (hunk 1 above)
Comment (xluo-aws) [reply public/pages/SplitIndex/components/SplitIndexFlyout/SplitIndexFlyout.tsx:55]: Added index name and replica help text to constants.ts. shard number help varies depending on operation, for example, split doesn't want to show message as default shard number is 1. So didn't add one common help text for shard number.

File: public/pages/SplitIndex/container/SplitIndex/SplitIndex.tsx (hunk 2 above)
Comment (xluo-aws) [reply public/pages/SplitIndex/container/SplitIndex/SplitIndex.tsx:47]: Updated in latest code

**Review by SuZhou-Joe — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Pull request merged**